### DISCUSSION TITLE
Layout issue on multiple pages

### DISCUSSION THREAD

**Discussion by haleygelato** [community/community#114662] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My video could have been better, but the video size limit is too strict. I tried refreshing, but it didn't fix it. This issue isn't limited to just the profile page; it affects other pages as well.

https://github.com/community/community/assets/112786962/88b9a758-b3d7-48a9-93ab-196ab1c30b67

**Comment by ethanwater** [comment]
Definitely not GitHub. Issues like these are usually caused by browser extensions or configurations that can lead to overflow.

**Reply by haleygelato** [reply]
>  Definitely not GitHub. Issues like these are usually caused by browser extensions or configurations that can lead to overflow.

That might make sense, but have not experienced this issue with other websites, not at the same time or at any other time. Also don’t think any of my extensions would interfere with the page layout.

**Reply by ethanwater** [reply]
Does this happen on any other sites? Also, have you tried to see if the same effect occurs when using other browsers? Extensions like ad blockers that change the behavior/appearance of websites can sometimes cause these errors. 

 I had an issue like this before I transitioned to Safari.

**Reply by haleygelato** [reply]
> Does this happen on any other sites? Also, have you tried to see if the same effect occurs when using other browsers? Extensions like ad blockers that change the behavior/appearance of websites can sometimes cause these errors. 
> 
> 
> 
>  I had an issue like this before I transitioned to Safari. 

^^I haven’t had this issue with other websites. Also, I don’t have ad blockers and haven’t tested it on other browsers. I was using Chrome and just wanted to post here in case others are experiencing the same thing. But since it’s working fine for me now, can’t really pin down what the issue was. But thanks a lot for weighing in on this!!

**Reply by ethanwater** [reply]
Ha! I didn't do much but thankfully it's gone. Computers can be silly sometimes.
